README.md

```diff
@@ -103,7 +105,7 @@ if (initialLocation) {
 
 ### Provided actions and state
 
-`redux-little-router` provides the following action creators for navigation:
+`redux-little-router` provides the following **action creators** for navigation:
```

Comment:
You'd think someone using this router would be extra familiar with redux and therefore not snag on this, but I think a lot of people new to redux will appreciate these couple of clarifications.